src/main/kotlin/Command/commands/ChooseNoodlesCommand.kt

```diff
@@ -0,0 +1,19 @@
+package Command.commands
+
+import Command.base.Cart
+import Command.base.Command
+
+class ChooseNoodlesCommand(val cart : Cart) : Command {
```

Comment:
You have delegate to cart the responsibility of print `chooseNoodles` it's okey, it's your decision 👍